.github/pull_request_template.md

```diff
@@ -24,6 +24,16 @@ Please ensure your pull request (PR) adheres to the following guidelines:
 - [ ] Documentation
 - [ ] Other (please describe): 
 
+## Security Considerations (REQUIRED)
```

Comment:
should we add "checkboxes" like the sections above this one have?

Reply:
I think that would make sense, will add them